cmd/crc/cmd/config/config.go

```diff
@@ -107,6 +108,13 @@ func RegisterSettings(cfg *config.Config) {
 		"Consent to collection of anonymous usage data (yes/no)")
 }
 
+func defaultNetworkMode() network.Mode {
+	if runtime.GOOS == "darwin" && version.IsMacosInstallPathSet() {
+		return network.VSockMode
+	}
+	return network.DefaultMode
```

Comment:
@guillaumerose what would be the value to 'revert' to the legacy (switched) network mode?

Reply:
`crc config set network-mode default` to use the same network mode as in the previous releases.

Reply:
and that would be a 'lie', as it is not the default. We might wanna consider changing the wording of this for macOS. WDYT?

Reply:
yes this is confusing, this is tracked in https://github.com/code-ready/crc/issues/1945
Imo, that's good enough for this release, and we improve this in the next release.